app/init.php

```diff
@@ -90,9 +90,12 @@
 Config::load('storage-inputs', __DIR__.'/config/storage/inputs.php'); 
 Config::load('storage-outputs', __DIR__.'/config/storage/outputs.php'); 
 
-Resque::setBackend(App::getEnv('_APP_REDIS_HOST', '')
-    .':'.App::getEnv('_APP_REDIS_PORT', ''));
-
+$auth = App::getEnv('_APP_REDIS_AUTH',null);
+if($auth != null) {
```

Comment:
Not sure default value is actually null. Docker might treat it as an empty string.

Reply:
oh, got it!